modin/backends/pandas/query_compiler.py

```diff
@@ -2583,24 +2630,54 @@ def groupby_dict_agg(self, by, func_dict, groupby_args, agg_args, drop=False):
             lambda df: df.groupby(by=by, **groupby_args).agg(func_dict, **agg_args)
         )
 
-    def groupby_agg(self, by, axis, agg_func, groupby_args, agg_args, drop=False):
-        # since we're going to modify `groupby_args` dict in a `groupby_agg_builder`,
+    def groupby_agg(
+        self,
+        by,
+        is_multi_by,
+        idx_name,
+        axis,
+        agg_func,
+        agg_args,
+        agg_kwargs,
+        groupby_kwargs,
+        drop_,
+        drop=False,
+    ):
+        agg_func = wrap_udf_function(agg_func)
+
+        if is_multi_by:
+            return self._default_to_pandas_groupby(
+                agg_func, by, axis, drop, groupby_kwargs, *agg_args, **agg_kwargs
+            )
```

Comment:
to be consistent we can replace it to
```python
return super().groupby_agg(...)
```
I've run tests locally with this change and they passed

Reply:
Looks like it makes sense.

Reply:
This change broke "default to pandas" warning message. Now we have strange message `UserWarning: <lambda> defaulting to pandas implementation.` that doesn't explain anything.